backend/src/main/java/com/ody/auth/dto/request/AppleAuthRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.ody.auth.dto.request;
+
+import com.ody.mate.domain.Nickname;
+import com.ody.member.domain.DeviceToken;
+import com.ody.member.domain.Member;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+
+public record AppleAuthRequest(
+
+        @Schema(description = "디바이스 토큰", example = "devicetokendevicetoken")
+        @NotNull
```

Comment:
Blank 관련 체크도 광범위하게 잡아주어도 좋을 것 같아요. null 뿐만 아니라 "", "   "에도 오류가 발생합니다.

```suggestion
        @NotBlank
```

Reply:
[반영] [25b375d](https://github.com/woowacourse-teams/2024-ody/pull/954/commits/25b375d85c26023d1b5acf7a144a45e83ee69774) (닉네임을 제외한 필드에 NotBlank를 적용했습니다)